deltachat-ffi/deltachat.h

```diff
@@ -6355,6 +6355,25 @@ void dc_event_unref(dc_event_t* event);
 
 #define DC_EVENT_CHATLIST_ITEM_CHANGED         2301
 
+/**
+ * Inform that the list of accounts has changed (account removed or added or (not yet implemented) the order changes)
+ *
+ * This event is only emitted by the account manager.
+ */
+
+#define DC_EVENT_ACCOUNTS_CHANGED              2302
+
+/**
+ * Inform that an account property that might be shown in the account list changed, namely:
+ * - is_configured
+ * - displayname
+ * - selfavatar
+ * - private_tag
+ * 
+ * This event is emitted from the account whose item it describes.
```

Comment:
```suggestion
 * This event is emitted from the account whose property changed.
```
(for me it's a more simple wording)